changelog.txt

```diff
@@ -2,6 +2,8 @@
  * Trying to access a deleted Realm object throw throws a proper IllegalStateException.
  * Added in-memory Realm support.
  * Closing realm on another thread different from where it was created now throws an exception.
+ * @Index annotation can also be applied to byte/short/int/long/boolean/Date
+ now.
```

Comment:
Strange, I checked for 10 times, no idea where does the new line come from. Will change this in the other PR.